Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1948,6 +1948,23 @@ extension BrowserViewController: SearchViewControllerDelegate {
     func searchViewController(_ searchViewController: SearchViewController, didHighlightText text: String, search: Bool) {
         self.topToolbar.setLocation(text, search: search)
     }
+    
+    func searchViewController(_ searchViewController: SearchViewController, shouldFindInPage query: String) {
+        topToolbar.leaveOverlayMode()
+        updateFindInPageVisibility(visible: true)
+        findInPageBar?.text = query
+        
+    }
+    
+    func searchViewControllerAllowFindInPage() -> Bool {
+        // Hides find in page for new tabs.
+        if let st = tabManager.selectedTab, let wv = st.webView {
```

Comment:
Cant you reduce this to one line:

`return tabManager.selectedTab?.webView?.url?.isAboutHomeURL == false`

Reply:
Copy pasted from 1.6 🤡